src/cljs/nr/gameboard/board.cljs

```diff
@@ -3,7 +3,7 @@
   (:require [cljs.core.async :refer [chan put! <! timeout] :as async]
             [clojure.string :as s :refer [capitalize includes? join lower-case split blank? starts-with? ends-with?]]
             [differ.core :as differ]
-            [game.core.card :refer [has-subtype? asset? rezzed? ice? corp?
+            [game.core.card :refer [condition-counter? get-title has-subtype? asset? operation? rezzed? ice? corp?
```

Comment:
Double check that this actually works lol. The card namespace can be accessed in the clojurescript but because we convert the game state to json, certain properties change from keywords to strings and get hecked up.

Reply:
This works fine for everything I've tested. Types are stored as strings and not keywords so I believe it doesn't run into the json conversion issue.